flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/SpillableSubpartitionTest.java

```diff
@@ -192,21 +198,39 @@ public void testConsumeSpilledPartition() throws Exception {
 
```

Comment:
nit: I'm evil I know, but could you replace mock `listener` with `new AwaitableBufferAvailablityListener()` if I please very nicely?

Reply:
done